flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "aarch64"]
```

Comment:
if that the only arch you support you don't need that file.

Reply:
I would prefer to keep it for clarity.

Reply:
the clarity is that since it is not there its easy to determine that it is build on all architectures. If you don't need it it should be there.

Reply:
It is not clear to me which architectures are supported by Flathub build system, e.g. I have seen 32-bit Intel and ARM processors mentioned in few places. Also, if support for a new architecture will be added by Flathub in future, I do not want packages to be automatically created for it.

Reply:
i386 and arm have been removed a couple of years ago. a new arch is unlikely to appear, and if it does it will be with a newer runtime. So nothing will happen magically.

Reply:
I removed flathub.json file.